kazoo/tests/test_connection.py

```diff
@@ -100,32 +99,126 @@ def back(state):
 
     def test_connection_write_timeout(self):
         client = self.client
-        ev = threading.Event()
+        ev_suspended = threading.Event()
```

Comment:
should `threading.Event` be `client.handler.event_object` instead to be gevent compatible?

Reply:
I think it should.  (I am acutely aware of that issue, as I made the mistake of copying a `time.sleep` statement from another test in a previous iteration—which deadlocked Travis :)

But: I did not want to introduce changes beyond what was necessary in this patch, and so have copied the "flow" from other tests.

What do you think?

@jeffwidman, @StephenSorriaux: Would you accept a subsequent "cleanup" patch for this?  If so, should it cover *all* the tests?  (That's going to be a bit noisy.)

Reply:
You made a good point. I would say this PR should only cover what is necessary for the patch, I would be fine with keeping what was currently present and open an issue (& PR) about updating all tests in order to better test gevent and other libs.

Reply:
Okay; noted!